misc/bash_completion.d/tarsnap

```diff
@@ -43,7 +43,8 @@ _tarsnap ()
 	# They won't be completed at all.
 	wotherarg="--checkpoint-bytes|--exclude|-f|--include|--maxbw|--maxbw-rate|
 		   |--maxbw-rate-down|--maxbw-rate-up|--newer|--newer-mtime|-s|
-		   |--strip-components|--disk-pause|--creationtime|--csv-file"
+		   |--strip-components|--disk-pause|--creationtime|--csv-file|
+		   |--progress-bytes|"
```

Comment:
I'm not sure what the syntax is here but it looks odd that there's a `|` at the end now and there wasn't before.

Reply:
I was wondering about that.  The new syntax works, and looks nicer to me... but the other files don't do it, so I've adjusted to keep that style.